android/.buildconfig.yaml

```diff
@@ -1,8 +1,8 @@
-libraryVersion: 0.2.0
+libraryVersion: 0.3.0
```

Comment:
I went with `0.3.0` to a have something unambiguously different for the Kotlin and Rust.